provision/vvv-init.sh

```diff
@@ -60,8 +60,24 @@ sed -i "s#{{DOMAINS_HERE}}#${DOMAINS}#" "${VVV_PATH_TO_SITE}/provision/vvv-nginx
 if [[ "${LIVE_URL}" != http* ]]; then
 	LIVE_URL=''
 else
-	LIVE_URL="\n\n    # Directives to send expires headers and turn off 404 error logging.\n    location ~* .(js|css|png|jpg|jpeg|gif|ico)$ {\n         expires 24h;\n         log_not_found off;\n         try_files \$uri \$uri\/ @production;\n    }\n\n    location @production {\n        resolver 8.8.8.8;\n        proxy_pass $LIVE_URL\/\$uri;\n    }\n"
+	LIVE_URL=$(cat <<'END_HEREDOC'
```

Comment:
Using a heredoc is a great idea! However this will break `sed` at least on my machine: ``sed: -e expression #1, char 45: unterminated `s' command``

Instead this seems to work: https://github.com/saulirajala/custom-site-template/blob/valu/provision/vvv-init.sh#L66. I think it's because those newlines needs to be escaped for `sed` to work properly.

Reply:
Is there a non-sed solution?

Reply:
AFAIK: no. But that doesn't necessarily mean anything 😄 

The problem here is to do kind of search and replace "proxy-images" code block to `vvv-nginx.conf`-file from `vvv-init.sh`-script. Using `sed` is the only solutions that I know for this kind of operations.